clang/test/SemaSYCL/unnamed-kernel.cpp

```diff
@@ -76,8 +73,7 @@ struct MyWrapper {
     });
 
 #ifndef __SYCL_UNNAMED_LAMBDA__
-    // expected-error@Inputs/sycl.hpp:220 {{'std::max_align_t' is an invalid kernel name type}}
-    // expected-note@Inputs/sycl.hpp:220 {{type 'std::max_align_t' cannot be in the "std" namespace}}
+    // expected-error@Inputs/sycl.hpp:220 {{'std::max_align_t' is an invalid kernel name, 'std::(anonymous)' is declared in the 'std' namespace}}
```

Comment:
This looks confusing.  Could we handle the 'anonymous' case differently?

Reply:
I don't think there's a whole lot more we could do, but I'm not worried about it being confusing. If a user adds an unnamed namespace to `std`, they've caused UB, so this shouldn't be a particularly common case for anyone to hit.